cmd/internal/server/handlers/configuration_form.go

```diff
@@ -86,6 +86,14 @@ func (ConfigurationForm) Handle(ctx context.Context, _ *fivetransdk.Configuratio
 					},
 				},
 			},
+			{
+				Name:        "starting_gtids",
+				Label:       "JSON containing keyspace, shard, and starting GTIDs",
+				Description: &tinyIntDesc,
+				Type: &fivetransdk.FormField_TextField{
+					TextField: fivetransdk.TextField_PlainText,
```

Comment:
Note on why this is a string:
 FiveTran does not support providing a JSON string field right now. These are the supported form fields: https://github.com/fivetran/fivetran_sdk/blob/main/common.proto#L18-L28